src/viur/core/tasks.py

```diff
@@ -478,11 +482,12 @@ def CallDeferred(func: t.Callable) -> t.Callable:
     def make_deferred(func, self=__undefinedFlag_, *args, **kwargs):
         # Extract possibly provided task flags from kwargs
         queue = kwargs.pop("_queue", "default")
+        call_deferred = kwargs.pop("_call_deferred", True)
```

Comment:
Can't these be directly put as arguments of `make_deferred`?

Reply:
I agree, that would be cleaner. But it's not done that way with the other arguments either. They should all be in the same style, otherwise it's a confusion. I would do this in a later refactoring.

Reply:
Therefore marked lines 484 to 485

Reply:
OK let's do this in another PR.

Reply:
Okay, I've opened #1130